FAQ.md

```diff
@@ -172,3 +172,17 @@ Steps:
 2. Restart APISIX
 
 Now you can trace the info level log in logs/error.log.
+
+## How to reload your own plugin
+
+The Apache APISIX plugin supports hot reloading. If your APISIX node has the Admin API turned on, then for scenarios such as adding / deleting / modifying plugins, you can hot reload the plugin by calling the HTTP interface without restarting the service.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/plugins/reload -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT
+```
+
+If your APISIX node does not open the Admin API, then you can manually load the plug-in by reloading openresty.
+
+```shell
+openresty -p /usr/local/apisix -c /usr/local/apisix/conf/nginx.conf -s reload
```

Comment:
`apisix reload` is better. use `CLI` is simpler.

Reply:
I originally wanted to use the `apisix reload` command, which ultimately triggered the process of `openresty reload nginx.conf`.

But I tried this command, I found that reload did not take effect, so I used `openresty -p /usr/local/apisix -c /usr/local/apisix/conf/nginx.conf -s reload`.